notebooks/hallucination_score_calculator.ipynb

```diff
@@ -0,0 +1,435 @@
+{
```

Comment:
<p>Why is there a risk here although the answer is basically the evidence provided in the prompt?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deepset-ai/haystack-cookbook/pull/252/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>